apps/web/public/static/locales/en/common.json

```diff
@@ -720,5 +726,15 @@
   "external_redirect_url": "https://example.com/redirect-to-my-success-page",
   "redirect_url_upgrade_description": "In order to use this feature, you need to upgrade to a Pro account.",
   "duplicate": "Duplicate",
-  "you_can_manage_your_schedules": "You can manage your schedules on the Availability page."
+  "you_can_manage_your_schedules": "You can manage your schedules on the Availability page.",
+  "request_reschedule_booking": "Request to reschedule your booking",
+  "reason_for_reschedule": "Reason for reschedule",
+  "book_a_new_time": "Book a new time",
+  "reschedule_request_sent": "Reschedule request sent",
+  "reschedule_modal_description": "This will cancel the scheduled meeting, notify the scheduler and ask them to pick a new time.",
+  "reason_for_reschedule_request": "Reason for reschedule request",
+  "send_reschedule_request": "Send reschedule request",
+  "edit_booking": "Edit booking",
+  "reschedule_booking": "Reschedule booking",
```

Comment:
![image](https://user-images.githubusercontent.com/3504472/163277994-67004d56-eb7a-43be-adbc-aebacb368d42.png)

These seems confusing IMO, why not, "Reschedule booking" and "Send reschedule request"?

Reply:
cc @Jaibles